docs/docs/getting-started/offline-mode.md

```diff
@@ -0,0 +1,9 @@
+Hydra supports an offline mode, which allows for disabling the Layer 1 interface (that is, the underlying Cardano blockchain which Hydra heads use to seed funds and ultimately funds are withdrawn to). Disabling Layer 1 interactions allows use-cases which would otherwise require running and configuring an entire Layer 1 private devnet.
+
+In this offline mode, only the Layer 2 ledger is run. Therefore, ledger genesis parameters that normally influence things like time-based transaction validation, may be set to defaults that aren't reflective of mainnet. To set this, set --ledger-protocol-parameters to a non-zero file. TODO: show how to obtain this.
```

Comment:
Would be great to resolve the TODO

Reply:
I think u not only have the l2 ledger but also:
- networking stack
- persistence

Would be really good to explain why did we maintain these 2 layers.

At the same time I would like to know:
"how fast an offline head runs and how many tx am I able process per unit of time."

Reply:
To resolve the TODO i think we can link to https://hydra.family/head-protocol/unstable/docs/configuration/#ledger-parameters

Reply:
We don't have those metrics yet but we'll obtain them soon and add them in

Reply:
additionally, when https://github.com/input-output-hk/hydra/pull/1413 is merged, I think it'll be good to mention here, wrt additional flexibility in changing how persistence works

Reply:
would it be possible to add benchmarking info in a later PR or should it be done in order to get this merged?